pulumi/manifest.json

```diff
@@ -0,0 +1,37 @@
+{
+  "display_name": "Pulumi",
+  "maintainer": "team@pulumi.com",
+  "manifest_version": "1.0.0",
+  "name": "pulumi",
+  "metric_prefix": "pulumi.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Infrastructure as code for any cloud using your favorite programming languages",
+  "guid": "58481132-c79e-4659-8064-7cdaabbbc981",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "PULUMI",
+  "categories": [
+    "cat-orchestration",
+    "cat-aws",
+    "cat-azure",
+    "cat-cloud",
+    "cat-google-cloud",
+    "cat-configuration-deployment"
```

Comment:
The categories here must be from the existing ones you can find here: https://docs.datadoghq.com/integrations/

Reply:
@FlorianVeaux I believe that I've addressed this (the docs are not very clear but  I've standardized on category names per your feedback).